src/components/landing-page/landing-page.controller.ts

```diff
@@ -2,10 +2,39 @@ import * as angular from 'angular';
 
 
 export class LandingPageController implements angular.IController {
+  static $inject = ['$scope', '$timeout'];
 
   public ctrl: any = this;
+  private $scope: any;
+  private $timeout: any;
+
+  constructor ($scope: any, $timeout: any) {
+    this.$scope = $scope;
+    this.$timeout = $timeout;
+  }
 
   public $onInit() {
     this.ctrl.searchText = '';
   }
+
+  public $postLink() {
+    this.$scope.$watch(() => {
+      return this.ctrl.alerts;
+    }, () => {
+      _.forEach(this.ctrl.alerts, (alert: any) => {
+        if (alert.type === 'success' && !alert.hidden) {
+          if (!alert.timerId) {
+            alert.timerId = this.$timeout(function () {
+              alert.timerId = -1;
+              if (!alert.isHover) {
```

Comment:
Nevermind, I see it will dismiss when I mouse out because it triggers an alert update